routes/index.js

```diff
@@ -67,14 +67,35 @@ router.get('/', async (req, res) => {
       } else {
         req.session.username = username
 
+        var subs = [
+          ['USERNAME', username],
+          ['REALNAME', realname],
+          ['LAB_TITLE', title],
+          ['LAB_DURATION_HOURS', config.eventHours],
+          ['LAB_USER_COUNT', config.accounts.number],
+          ['LAB_USER_PASS', config.accounts.password],
+          ['LAB_USER_ACCESS_TOKEN', config.accounts.accessToken],
+          ['LAB_USER_PREFIX', config.accounts.prefix],
+          ['LAB_ADMIN_PASS', config.adminPassword],
```

Comment:
@jamesfalkner @Crosbie we should remove this. Is there a good reason that the administrator password and session secret could be injected into a plaintext URL that the lab users can see? it's for labs so it's low stakes, but is still unwise.

Otherwise this PR really useful 👍